docs/setup-sensor/docker-multiple-connections.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Measuring multiple connections
```

Comment:
I think maybe "multiple interfaces" is a better framing/title?
Otherwise it kinda leaves me wondering "oh, how?" until reading quite a ways in and finding "oh by using multiple interfaces"

Reply:
Fair point. I changed the title(s) and filename